src/lightning/pytorch/trainer/setup.py

```diff
@@ -72,7 +72,7 @@ def _init_debugging_flags(
 
         trainer.limit_test_batches = num_batches
         trainer.limit_predict_batches = num_batches
-        trainer.fit_loop.max_steps = num_batches
+        trainer.fit_loop.epoch_loop.max_steps = num_batches
```

Comment:
We can do this because the value gets already checked on line 54. So the setter is redundant.